src/community/blogs-talks-podcasts/index.md.njk

```diff
@@ -0,0 +1,74 @@
+---
+title: Blog posts, videos and podcasts
+description: See blog posts, video talks, and podcasts about the GOV.UK Design System and Prototype Kit
+section: Community
+layout: layout-pane.njk
+order: 8
+---
+
+See blog posts, video talks, and podcasts about the GOV.UK Design System and Prototype Kit that our team and community have shared.
```

Comment:
```suggestion
To learn more about the GOV.UK Design System and Prototype Kit, 
check out these blog posts, video talks and podcasts that our team and community have shared.
```

Reply:
Suggesting this as maybe it would be good to say to users why they should engage with our blogs, videos and podcasts.